python_examples/basic_infection_history_simulation/simulation_outputs/output.csv

```diff
@@ -1,62 +1,62 @@
 time,InfectionStatus.Susceptible,InfectionStatus.Exposed,InfectionStatus.InfectASympt,InfectionStatus.InfectMild,InfectionStatus.InfectGP,InfectionStatus.InfectHosp,InfectionStatus.InfectICU,InfectionStatus.InfectICURecov,InfectionStatus.Recovered,InfectionStatus.Dead,InfectionStatus.Vaccinated

-0,90,0,0,10,0,0,0,0,0,0,0

-1.0,90,0,0,8,0,0,0,0,2,0,0

-2.0,75,17,0,8,0,0,0,0,0,0,0

-3.0,67,21,2,9,1,0,0,0,0,0,0

-4.0,66,20,3,7,2,0,0,0,2,0,0

-5.0,68,18,5,7,2,0,0,0,0,0,0

-6.0,66,14,8,10,1,0,0,0,1,0,0

-7.0,66,14,8,8,0,0,0,0,4,0,0

-8.0,67,15,10,6,0,0,0,0,2,0,0

-9.0,65,16,11,6,0,0,0,0,2,0,0

-10.0,66,16,12,4,0,0,0,0,2,0,0

-11.0,66,14,13,4,2,0,0,0,1,0,0

-12.0,66,13,15,4,2,0,0,0,0,0,0

-13.0,63,14,14,5,2,0,0,0,2,0,0

-14.0,65,12,13,6,3,0,0,0,1,0,0

-15.0,63,12,14,5,5,0,0,0,1,0,0

-16.0,62,13,13,4,6,0,0,0,2,0,0

-17.0,64,10,13,5,4,0,0,0,4,0,0

-18.0,68,6,12,4,7,0,0,0,3,0,0

-19.0,67,6,11,6,7,1,0,0,2,0,0

-20.0,65,9,11,6,7,1,0,0,1,0,0

-21.0,63,10,12,7,6,1,0,0,1,0,0

-22.0,64,8,12,7,5,1,0,0,3,0,0

-23.0,67,4,11,10,6,1,0,0,1,0,0

-24.0,67,5,8,8,5,2,0,0,5,0,0

-25.0,70,6,9,8,2,2,0,0,3,0,0

-26.0,72,5,10,9,2,2,0,0,0,0,0

-27.0,72,4,11,7,2,2,0,0,2,0,0

-28.0,70,4,10,6,5,2,0,0,3,0,0

-29.0,71,5,10,5,6,1,0,0,2,0,0

-30.0,69,8,9,5,6,1,0,0,2,0,0

-31.0,68,9,7,6,6,1,0,0,3,0,0

-32.0,70,6,8,6,5,1,0,0,4,0,0

-33.0,74,4,9,6,5,1,0,0,1,0,0

-34.0,74,3,7,6,3,2,0,0,5,0,0

-35.0,77,4,7,7,3,2,0,0,0,0,0

-36.0,72,7,6,8,4,2,0,0,1,0,0

-37.0,72,6,7,9,4,1,0,0,1,0,0

-38.0,73,5,6,9,5,1,0,0,1,0,0

-39.0,72,7,6,7,4,0,0,0,3,1,0

-40.0,74,7,6,8,2,0,0,0,2,1,0

-41.0,75,7,5,7,3,0,0,0,2,1,0

-42.0,77,5,5,7,3,0,0,0,2,1,0

-43.0,78,5,5,7,3,0,0,0,1,1,0

-44.0,77,5,6,6,3,0,0,0,2,1,0

-45.0,76,8,6,5,3,0,0,0,1,1,0

-46.0,75,8,7,4,2,0,0,0,3,1,0

-47.0,78,5,8,6,2,0,0,0,0,1,0

-48.0,76,6,8,5,3,0,0,0,1,1,0

-49.0,77,2,8,6,4,0,0,0,2,1,0

-50.0,78,3,8,6,3,0,0,0,1,1,0

-51.0,79,2,8,5,3,0,0,0,2,1,0

-52.0,80,2,6,2,4,0,0,0,5,1,0

-53.0,85,2,6,0,3,0,0,0,3,1,0

-54.0,87,2,6,1,3,0,0,0,0,1,0

-55.0,84,5,6,1,3,0,0,0,0,1,0

-56.0,83,6,6,1,3,0,0,0,0,1,0

-57.0,83,5,5,2,3,0,0,0,1,1,0

-58.0,84,5,3,2,2,0,0,0,3,1,0

-59.0,87,3,3,4,2,0,0,0,0,1,0

-60.0,87,2,4,3,2,0,0,0,1,1,0

+0,99,0,0,1,0,0,0,0,0,0,0

+1.0,99,0,0,1,0,0,0,0,0,0,0

+2.0,99,0,0,1,0,0,0,0,0,0,0

+3.0,99,0,0,1,0,0,0,0,0,0,0

+4.0,99,0,0,1,0,0,0,0,0,0,0

+5.0,99,0,0,0,0,0,0,0,1,0,0

+6.0,99,0,0,0,0,0,0,0,1,0,0

+7.0,99,0,0,0,0,0,0,0,1,0,0

+8.0,99,0,0,0,0,0,0,0,1,0,0

+9.0,99,0,0,0,0,0,0,0,1,0,0

+10.0,99,0,0,0,0,0,0,0,1,0,0

+11.0,99,0,0,0,0,0,0,0,1,0,0

+12.0,99,0,0,0,0,0,0,0,1,0,0

+13.0,99,0,0,0,0,0,0,0,1,0,0

+14.0,99,0,0,0,0,0,0,0,1,0,0

+15.0,99,0,0,0,0,0,0,0,1,0,0

+16.0,99,0,0,0,0,0,0,0,1,0,0


```

Comment:
Did you mean to commit an updated version of this file? It seems to be a simulation where the pandemic doesn't take off and doesn't exemplify this software like the previous one. Might be best to revert these changes.

Reply:
Thanks for catching this, and no this was a mistake! (same for all the files you mentioned below)